plugin/src/main/groovy/com/here/gradle/plugins/jobdsl/tasks/AbstractDslTask.groovy

```diff
@@ -63,6 +63,11 @@ abstract class AbstractDslTask extends JavaExec {
         main = mainClass
         classpath = project.sourceSets.main.runtimeClasspath + project.buildscript.configurations.classpath +
                 project.configurations.jenkinsPlugins
+        jvmArgs = [
+                // Uncomment the following two lines for debugging the exec tasks.
+                //'-Xdebug',
+                //'-Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=5005'
```

Comment:
FYI, this string does not match with what IntelliJ suggests, which is

    -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005

for Java 5 - 8, and

    -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005

for Java 9+.

Reply:
Done.